documentation/functions/domain/DefaultTTL.md

```diff
@@ -7,7 +7,7 @@ parameter_types:
 ---
 
 DefaultTTL sets the TTL for all records in a domain that do not explicitly set one with [TTL](../record/TTL.md). If neither `DefaultTTL` or `TTL` exist for a record,
-it will use the DNSControl global default of 300 seconds.
+it will use the DNSControl global default of 300 seconds. NS records are currently a special case, and ignore DefaultTTL. To anchor those to a default TTL, use e.g. `{"ns_ttl":"3600"},`.
```

Comment:
Rather than `{"ns_ttl":"3600"}` the user can use NAMESERVER_TTL("3600").

How about:  

```
NS records are currently a special case, and ignore DefaultTTL. The default TTL for NS
records is set with `NAMESERVER_TTL()`.
```

Reply:
re-done